src/osd/OSD.cc

```diff
@@ -3200,16 +3200,18 @@ int OSD::shutdown()
   cct->get_admin_socket()->unregister_command("ops");
   cct->get_admin_socket()->unregister_command("dump_blocked_ops");
   cct->get_admin_socket()->unregister_command("dump_historic_ops");
-  cct->get_admin_socket()->unregister_command("dump_historic_ops_by_duration");
   cct->get_admin_socket()->unregister_command("dump_historic_slow_ops");
+  cct->get_admin_socket()->unregister_command("dump_historic_ops_by_duration");
```

Comment:
i don't think i am asking any questions here? and i am trying to clarify my intention.